tests/services/order_validation_service/test_case_sample_rules.py

```diff
@@ -47,6 +82,40 @@ def test_validate_tube_container_name_unique(valid_order: OrderWithCases):
     assert errors[0].sample_index == 0 and errors[0].case_index == 0
 
 
+def test_applications_exist(valid_order: OrderWithCases, base_store: Store):
+    # GIVEN an order where one of the samples has an invalid application
+    for case in valid_order.cases:
+        case.samples[0].application = "Invalid application"
+
+    # WHEN validating the order
+    errors = validate_application_exists(order=valid_order, store=base_store)
+
+    # THEN an error should be returned
+    assert errors
+
+    # THEN the error should be about the ticket number
+    assert isinstance(errors[0], ApplicationNotValidError)
+
+
+def test_applications_not_archived(
```

Comment:
what is the difference between this test and the previous one? Seems like you can only keep this one

Reply:
They are testing different rules. One is whether the application tag matches anything in the database. This test makes sure that the application is not archived for new samples